src/script.js

```diff
@@ -353,7 +341,7 @@ function calculateExpenses() {
     const odjSk_selectVal = document.querySelector('input[name="odjSk"]:checked').value;
     const nacPrik_selectVal = document.querySelector('input[name="nacPrik"]:checked').value;
     const napNivo_selectVal = document.querySelector('input[name="napNivo"]:checked').value;
-    
+
 
     // Izračunaj stroške omrežnine (glede na izbrano odjemno skupino) - stroški starega načina omrežnine
     // Stroški omrežnine v primeru izbire napetostnega nivoja NN
```

Comment:
Is really strucutre of price_m_obr (array of arrays of arrays) the best way if you are then using [0][0][0] ? i would rather maybe user object of arrays?